### PR TITLE
Adds 'published by' column to file view

### PR DESCRIPTION
**Pull request by Tschuppi81**
Files: Adds 'published until' column to file view

TYPE: Feature
LINK: ogc-1696

## Checklist

- [x] I have performed a self-review of my code
- [x] I considered adding a reviewer
- [x] I have updated the PO files
- [ ] I made changes/features for both org and town6 -> org does not have this feature
- [x] I have tested my code thoroughly by hand
- [x] I have added tests for my changes/features

### CODE REVIEW

File: src/onegov/town6/templates/files.pt [hunk 1]
```diff
@@ -70,6 +70,16 @@
...
                                        i18n:translate>Upload Date</a>
                                 </th>
+                                <th>
+                                    <span tal:condition="model.order_by == 'publish_end_date'">
+                                        ${group}
+                                    </span>
+                                    <a tal:condition="model.order_by != 'publish_end_date'"
+                                       href="${request.link(model.for_order('publish_end_date'))}"
+                                       i18n:translate>
+                                       Published by
```
Comment (BreathingFlesh) [inline src/onegov/town6/templates/files.pt:80]: This needs a translation. Currently it's just displayed in english.

**Review by BreathingFlesh — COMMENTED** [review]
To me it's not really clear how the published by date differs from the upload date. Except the time which seems to be 0:00 instead of the upload time. I don't know how to manually change the published by date.

**Comment by Tschuppi81** [comment]
Published by is the date until a document may be published
image

**Comment by Tschuppi81** [comment]
I corrected the translation to `published until`

**Pull request merged**